app/src/main/java/com/keylesspalace/tusky/components/viewthread/ViewThreadFragment.kt

```diff
@@ -178,7 +178,11 @@ class ViewThreadFragment :
                         threadProgressBar = getProgressBarJob(binding.threadProgressBar, 500)
                         threadProgressBar.start()
 
-                        adapter.submitList(listOf(uiState.statusViewDatum))
+                        if (viewModel.isInitialLoad) {
+                            adapter.submitList(listOf(uiState.statusViewDatum))
+
+                            // else this "submit one and then all on success below" will always center on the one
+                        }
```

Comment:
NOTE / question the `scrollToPositionWithOffset` below in success is normally not necessary. The "submit 1" here results in exactly that.

However I guess that is a safe-guard. This location here ("ThreadLoad") is maybe not called always or not always before?